components/lib/datatable/DataTable.js

```diff
@@ -369,26 +369,30 @@ export const DataTable = React.forwardRef((inProps, ref) => {
             }
 
             if (ObjectUtils.isNotEmpty(widths)) {
-                createStyleElement();
+                addInnerHtml(widths);
+            }
+        }
+    };
 
-                let innerHTML = '';
-                let selector = `.p-datatable[${attributeSelector.current}] > .p-datatable-wrapper ${isVirtualScrollerDisabled() ? '' : '> .p-virtualscroller'} > .p-datatable-table`;
+    const addInnerHtml = (widths) => {
```

Comment:
Call it `addColumnWidthStyles`

Reply:
Also looks like there is LINT issue.

```
./components/lib/datatable/DataTable.js
730:17  Error: Expected blank line before this statement.  padding-line-between-statements
```

Reply:
Hey @melloware, calling it `addColumnWidthStyles` as per your comment and fixed lint issues.